csaf_2.0/examples/CVE-2018-0171-modified.json

```diff
@@ -53,7 +53,9 @@
       "initial_release_date": "2018-03-28T16:00:00Z",
       "current_release_date": "2018-04-17T15:08:41Z",
       "generator": {
-        "engine": "TVCE"
```

Comment:
I have to come up with a better name and structure for the Cisco engine in our (Cisco) end. Other than that, this looks great! If we decide to change it, I will update the example.